polymer-punk-brew/src/punk-brew-app/punk-brew-app.html

```diff
@@ -75,18 +73,6 @@
           }
         };
       }
-
-      static get observers() {
-        return [
-          '_routePageChanged(routeData.page, subrouteData.id)',
-        ];
-      }
-
-      _routePageChanged(page, id) {
```

Comment:
It was unnecessary. If the value is undefined it will automatically use the route defined in the fallback-selection attribute on iron-pages. Which is already set to "home".

https://www.webcomponents.org/element/PolymerElements/iron-pages/elements/iron-pages#property-fallbackSelection